cwltool/cwlviewer.py

```diff
@@ -0,0 +1,137 @@
+import rdflib
+from urllib.parse import urlparse
+import os
+import pydot  # type: ignore
+
+
+class CWLViewer:
+    """
+    Visualize a CWL workflow. The viewer tagrets to produce similar images with the
+    https://github.com/common-workflow-language/cwlviewer.
+    """
```

Comment:
cwltool/cwlviewer.py:8: D205: 1 blank line required between summary line and description (found 0)
cwltool/cwlviewer.py:8: D400: First line should end with a period (not 'e')